src/mesh/Draco_Mesh.cc

```diff
@@ -256,21 +243,81 @@ void Draco_Mesh::compute_cell_to_cell_linkage(
       Check(sides_in_common.size() <= 1);
       if (sides_in_common.size() > 0) {
         // populate cell-to-side linkage
-        // TODO: replace with typedef Draco_Layout::Boundary_Layout
         cell_to_side_linkage[cell].push_back(
             std::make_pair(sides_in_common[0], vec_node_vec[l]));
       }
 
-      // TODO: add check for ghost face
+      // check if this vertex pair has a ghost cell
+      // \todo: add DbC to ensure these are sorted
+      const std::vector<unsigned> &vert0_ghosts =
+          node_to_ghost_cell_map[vec_node_vec[l][0]];
+      const std::vector<unsigned> &vert1_ghosts =
+          node_to_ghost_cell_map[vec_node_vec[l][1]];
+
+      // find common ghost cells
+      std::vector<unsigned> ghost_cells_in_common;
+      std::set_intersection(vert0_ghosts.begin(), vert0_ghosts.end(),
+                            vert1_ghosts.begin(), vert1_ghosts.end(),
+                            std::back_inserter(ghost_cells_in_common));
```

Comment:
I didn't know this function existed! Cool.

Reply:
It should be noted that this complexity is achieved as a result of the vector arguments being sorted, by construction.